
uint8 id

uint16 open_position
uint16 open_speed
uint16 open_hold_torque
---
uint8 state
